prioritizer/prioritizer.py

```diff
@@ -5,14 +5,16 @@
 import re
 import json
 import pandas as pd
+from pydantic import BaseModel
+from typing import List
 
 load_dotenv()
 
 api_key = os.getenv("OPENAI_API_KEY")
 client = OpenAI(api_key=api_key)
 
 # Constants for quantitative scoring
```

Comment:
For the declared constants, a comment or __doc__ string would be good to explain these values.
E.g. why MAX_EMISSIONS_REDUCTIONS = 500000 or MAX_COST = 60000000 and so on